hoomd/md/pair.py

```diff
@@ -227,24 +227,31 @@ class LJ(_Pair):
         \end{eqnarray*}
 
     See :py:class:`_Pair` for details on how forces are calculated and the
-    available energy shifting and smoothing modes.  Use ``params`` dictionary
-    to set potential coefficients.
-
-    The following coefficients must be set per unique pair of particle types:
-
-    - :math:`\varepsilon` - *epsilon* (in energy units)
-    - :math:`\sigma` - *sigma* (in distance units)
-    - :math:`r_{\mathrm{cut}}` - *r_cut* (in distance units)
-      - *optional*: defaults to the global r_cut specified in the pair command
-    - :math:`r_{\mathrm{on}}`- *r_on* (in distance units)
-      - *optional*: defaults to the global r_on specified in the pair command
+    available energy shifting and smoothing modes.  Use `params` dictionary
+    to set potential coefficients. The coefficients must be set per 
+    unique pair of particle types.
+    
+    Attributes:
+        params (TypeParameter[``particle_types``, dict]):
```

Comment:
```suggestion
        params (TypeParameter[tuple[``particle_types``, ``particle_types``], dict]):
```
This needs to be changed for the other potentials as well.

Reply:
Perhaps:
```suggestion
        params (TypeParameter[tuple[``particle_type``, ``particle_type``], dict]):
```